docs/mcp_user_guide.md

```diff
@@ -0,0 +1,178 @@
+# AWS Deadline Cloud MCP Server User Guide
+
+## Architecture Overview
+
+The AWS Deadline Cloud MCP Server uses FastMCP within the MCP Python SDK to provide a simplified, decorator-based approach to tool registration while maintaining full MCP protocol compliance.
```

Comment:
This doc doesn't define the MCP acronym, or what the purpose of an MCP is. That would be great to describe in this section, along with an overview of what you can accomplish when you use this MCP.